lib/collection/src/collection_manager/segments_searcher.rs

```diff
@@ -254,10 +307,9 @@ impl SegmentsSearcher {
                         search_in_segment(
                             segment,
                             partial_batch_request,
-                            0,
```

Comment:
We used to pass explicitly `zero` in that case vs `available_point_count` for the query above.

Now both `search_in_segment` will use the same value from the `query_context`.

Not sure of the consequences yet.

Reply:
None, if the heuristic is not used, it won't use this parameter